<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super TIN-TIN</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>204</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="pushSzukajZnajomych">
       <property name="text">
        <string>Szukaj znajomych</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listaZnajomych"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="titListaZnajomych">
       <property name="text">
        <string>Lista znajomych</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushDodaj">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushUsun">
         <property name="text">
          <string>Usuń</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushGrupRozmawiaj">
       <property name="text">
        <string>Rozmowa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushWyslijPlik">
       <property name="text">
        <string>Wyslij plik</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushGrupWyslijPlik">
       <property name="text">
        <string>Grupowe wysyłanie</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="logoView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progresWysyl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wysyłanie pliku :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Odbieranie pliku :</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progresOdbior">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionSzukajZnajomych"/>
    <addaction name="separator"/>
    <addaction name="actionWyloguj"/>
    <addaction name="actionZakoncz"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSzukajZnajomych">
   <property name="text">
    <string>Szukaj znajomych</string>
   </property>
  </action>
  <action name="actionZakoncz">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionWyloguj">
   <property name="text">
    <string>Wyloguj</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
